eslint/noBuffer.js

```diff
@@ -0,0 +1,22 @@
+// @ts-no-check
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  create(context) {
+    return {
+      Identifier: function (node) {
+        if (node.name === 'Buffer') {
+          if (
+            (node.parent.property.type === 'Identifier' && node.parent.property.name === 'from') ||
+            (node.parent.property.type === 'Identifier' &&
+              node.parent.property.name.includes('alloc'))
```

Comment:
We certainly can.  I just wanted to be careful and not make it too broad in case there was some other obscure use of the keyword `Buffer` I wasn't thinking of.  I think you're right though since it makes sense so we also preclude any parameter/return types of Buffer as well.